solutions/A->B.js

```diff
@@ -0,0 +1,42 @@
+const INPUT_PATH = "../inputs/A->B.txt";
+
+const fs = require("fs");
+const filePath = process.platform === "linux" ? "/dev/stdin" : INPUT_PATH;
+const input = fs.readFileSync(filePath).toString().trim().split("\n");
+
+function solution(inputArguments) {
+  const [start, goal] = inputArguments[0].split(" ").map(Number);
+  const queue = [[start, 1]];
+  const visited = { start: true };
+
+  if (start === goal) return 1;
```

Comment:
```suggestion
  if (start === goal) {
    return 1;
  }
```

조건문 컨벤션이 하나로 통일되면 좋을 것 같습니다!! 👍